matchms/importing/load_from_msp.py

```diff
@@ -149,16 +147,20 @@ def parse_metadata(rline: str, params: dict):
     matches = []
     splitted_line = rline.split(":", 1)
     if splitted_line[0].lower() == 'comments' and "=" in splitted_line[1]:
-        pattern = r'(\S+)="([^"]*)"|"(\w+)=([^"]*)"|"([^"]*)=([^"]*)"|(\S+)=(\d+(?:\.\d*)?)'
+        pattern = r'(\S+)="([^"]+)"|' \
```

Comment:
I have changed this to require a symbol in both groups (before and after =) before cases like. "compound_name=" would make the code crash, since the second part is None. But we might want to store this as compound_name: None in the spectrum object. This long regex seems a bit hacky and I think it can be simplified, but I am afraid to remove edge cases that cannot be detected in that case anymore, what do you think?